docs/authenticate.js

```diff
@@ -204,8 +204,9 @@ class Authenticator {
             alert(
                 'Error contacting server for access without EarthEngine ' +
                 'whitelisting. Please reload page and log in with an ' +
-                'EarthEngine-whitelisted account or contact website ' +
-                'maintainers with error from JavaScript console.');
+                'EarthEngine-whitelisted account or contact ' +
+                'technology.manager@givedirectly.org ' +
```

Comment:
nit: could probably fit a few words from next line on this line, but doesn't change line count, so NBD